ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/ptp.py

```diff
@@ -54,6 +54,11 @@ def ptp_enabled(self: SharedUtils) -> bool:
         default_ptp_enabled = get(self.hostvars, "ptp.enabled")
         return get(self.switch_data_combined, "ptp.enabled", default=default_ptp_enabled) is True
 
+    @cached_property
+    def mlag_ptp_enabled(self: SharedUtils) -> bool:
+        default_mlag_ptp = get(self.hostvars, "mlag_ptp")
+        return get(self.switch_data_combined, "mlag_ptp", default=default_mlag_ptp) is True
```

Comment:
You have not added a schema for a global var.
```suggestion
        return get(self.switch_data_combined, "mlag_ptp") is True
```